torchreid/integration/sc/openvino_task.py

```diff
@@ -95,59 +90,34 @@ def __init__(
             Good value is the number of available cores. Defaults to 1.
         :param device: Device to run inference on, such as CPU, GPU or MYRIAD. Defaults to "CPU".
         """
-        super().__init__(model_file, weight_file, device, num_requests)
         self.labels = labels
-        self.input_blob_name = 'data'
-        self.n, self.c, self.h, self.w = self.net.input_info[self.input_blob_name].tensor_desc.dims
-        self.keep_aspect_ratio_resize = False
-        self.pad_value = 0
-        self.multilabel = multilabel
         self.empty_label = empty_label
-
-    @staticmethod
-    def resize_image(image: np.ndarray, size: Tuple[int], keep_aspect_ratio: bool = False) -> np.ndarray:
-        if not keep_aspect_ratio:
-            img = PIL.Image.fromarray(image).resize(size, resample=PIL.Image.BILINEAR)
```

Comment:
Classification models are trained with PILL resize, so expect an accuracy drop if `Classification(ImageModel)` is replacing it with cv.resize.